api/v1alpha1/stresschaos_types.go

```diff
@@ -119,9 +119,6 @@ type StressInstance struct {
 	// UID is the instance identifier
 	// +optional
 	UID string `json:"uid"`
-	// StartTime specifies when the instance starts
-	// +optional
-	StartTime *metav1.Time `json:"startTime"`
```

Comment:
Why delete `StartTime`? This `StartTime` was used to avoid the PID was reused.

Reply:
Oops. It's really a mistake. Checking whether the target is the child process of chaos-daemon cannot replace this start-time check.

I will fix it right now.